app/services/provider_interface/magic_link_authentication.rb

```diff
@@ -3,15 +3,19 @@ module MagicLinkAuthentication
     TOKEN_DURATION = 1.hour
```

Comment:
Do we still need this?

Reply:
Sorry, I think I'm missing something. Why don't we need this?